src/components/ProjectMetaDataTable/ProjectDetailsComponent.tsx

```diff
@@ -5,14 +5,14 @@ import { ProjectCollaborators } from "../ProjectCollaborators";
 import ProjectDetailsCard from "../ProjectDetailsCard";
 import { ProjectDetailsComponentProps } from "@/types/const";
 import UploadButton from "../Project/UploadButton";
+import { useProjectProvider } from "@/context/ProjectProvider";
 
 const ProjectDetailsComponent = ({ project }: ProjectDetailsComponentProps) => {
+  const { name, description } = useProjectProvider();
+
   return (
     <div className="flex flex-col items-center lg:items-start gap-2 w-full">
-      <ProjectDetailsCard
-        projectDescription={String(project?.description)}
-        projectName={String(project?.name)}
-      />
+      <ProjectDetailsCard projectName={name} projectDescription={description} />
```

Comment:
if both `name` and `description` are from a provider, couldn't you just import these variables, from the hook, directly in the `ProjectDetailsCard` component?
- remove `props`
- move usage of `useProjectProvider` into the `ProjectDetailsCard` component.

Reply:
This was already implemented here. I will change that. @SweetmanTech